addons/mrp/static/src/js/mrp.js

```diff
@@ -122,7 +122,17 @@ var TimeCounter = AbstractField.extend({
         } else {
             clearTimeout(this.timer);
         }
-        this.$el.html($('<span>' + moment.utc(this.duration).format("HH:mm:ss") + '</span>'));
+
+        function padStart(number){
+            return number.toString().padStart(2, '0')
+        }
+
+        const duration = moment.duration(this.duration, 'milliseconds');
+        const hours = Math.floor(duration.asHours());
+        const minutes = Math.floor(duration.asMinutes()) - hours * 60;
+        const seconds = Math.floor(duration.asSeconds() - (hours * 3600) - (minutes * 60));
+        
+        this.$el.html($('<span>', {text: padStart(hours) + ':' + padStart(minutes) + ':' + padStart(seconds)}, '</span>'));
```

Comment:
```suggestion
        this.$el.html($('<span>', {text: _.str.sprintf("%02d:%02d:%02d", hours, minutes, seconds}));
```
seems more readable. Also the closing tag is incorrect. `$('<span>')` will parse the tag then associate the properties with the resulting element.

Reply:
Also you may want `.empty().append()` here to avoid the security warning. It's a bit longer but it's more restrictive (as long as it's not used with raw strings anyway).